src/index.ts

```diff
@@ -141,7 +148,7 @@ export class ExtensionPortStream extends Duplex {
           // if the error is about message size being too large
           // note: this message doesn't currently happen on firefox, as it doesn't
           // have a maximum message size
-          err.message === 'Message length exceeded maximum allowed length.'
+          ExtensionPortStream.ErrorMessages.includes(err.message as typeof ExtensionPortStream.ErrorMessages[number])
```

Comment:
nit: `find` usually infers narrowed, literal types where `includes` doesn't. This is definitely uglier though.

```suggestion
          ExtensionPortStream.ErrorMessages.find(
            (errorMessage) => errorMessage === err.message,
          ) !== undefined
```

Reply:
Good to know. I'm ok with it as is myself.